Formula/ice.rb

```diff
@@ -14,6 +15,11 @@ class Ice < Formula
   depends_on "lmdb"
   depends_on "mcpp"
 
+  patch do
```

Comment:
Can you add a comment to the formula file above this line that explains what the patch does, so that maintainers can quickly see what the purpose of the patch is?

Might also be worth adding that it should be removed on the next release.